frontend/scripts/componentSchemas/run.ts

```diff
@@ -31,15 +31,21 @@ const addProperties = (propertyKeys: string[]) => {
   allPropertyKeys.push(...propertyKeys);
 };
 
-const generateComponentSchema = (name: string, layoutSchema: any) => {
+const generateComponentSchema = (name: string, layoutSchema: any, version: string) => {
   const definitionName = `Comp${name}`;
   console.log('definitionName: ', definitionName);
   const componentSchema = layoutSchema.definitions[definitionName];
   let schema: any = {
-    $id: `https://altinncdn.no/schemas/json/component/${name}.schema.v1.json`,
+    $id: `https://altinncdn.no/schemas/json/component/${name}.schema.v1.json`, // These links respond with 404?
```

Comment:
The only component that is available within the `components` folder is `number-format.schema.v1.json`, all other names would return 404 form CDN.

Reply:
Ok, I'll make this a discussion point for the new issue,